lib/bloc.dart

```diff
@@ -175,8 +175,7 @@ class SettingsBloc {
       }
     } else if (Platform.isIOS) {
       final ios = await deviceInfo.iosInfo;
-      // TODO: check what systemVersion returns on an actual iOS device.
-      if (int.tryParse(ios.systemVersion, radix: 10) >= 13 /* iOS 13*/) {
+      if (double.tryParse(ios.systemVersion) >= 13.0 /* iOS 13.0 */) {
```

Comment:
This will probably never happen but `double.tryParse` will return `null` if it could not parse the number. And `null >= 13.0` will throw an exception.

Reply:
It actually threw these exceptions on iOS when it was int.tryParse instead of double.tryParse. Which also created a never ending Circular Progress Indicator on the settings page where the dropdown menu for the "preferred theme" is.

Should I surround the if-condition with a try-catch where it does simply nothing if it catches an exception (= return false)?

Reply:
I was thinking about saving the result of `double.tryParse()` to a variable and extending the if to `if (version != null && version >= 13.0)`

Maybe crash the app if the version is `null` and we are running in debug mode.